src/services/NotificationsSchedulerService.js

```diff
@@ -44,12 +44,12 @@ async function getProjectWithId (projectId) {
 
 /**
  * extract the members of projects and build recipients list out of them
- * we can use `userId` to identify recipients
+ * we can use `email` to identify recipients
  * @param project the project
  * @returns {string[]} array of recipients
  */
 function buildProjectTeamRecipients (project) {
-  const recipients = _.unionBy(_.map(project.members, m => _.pick(m, 'userId')), 'userId')
+  const recipients = _.unionBy(_.map(project.members, m => _.pick(m, 'email')), 'email')
```

Comment:
@yoution Emails could be sent by `userId`. I'm not sure that project members would always have `email` addresses. Can we keep `userId` here, or this breaks something?